[2015-12-21 18:35:58] app.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException: No route found for "GET /index.html" (from "http://admin.portal.dev/") (uncaught exception) at /var/www/admin-portal/vendor/symfony/http-kernel/EventListener/RouterListener.php line 123 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /index.html\" (from \"http://admin.portal.dev/\") at /var/www/admin-portal/vendor/symfony/http-kernel/EventListener/RouterListener.php:123, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/index.html\". at /var/www/admin-portal/vendor/symfony/routing/Matcher/UrlMatcher.php:98)"} []
[2015-12-21 18:39:40] app.CRITICAL: RuntimeException: Unable to create the cache directory (true/fd). (uncaught exception) at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Cache/Filesystem.php line 60 {"exception":"[object] (RuntimeException(code: 0): Unable to create the cache directory (true/fd). at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Cache/Filesystem.php:60)"} []
[2015-12-21 18:40:51] app.CRITICAL: RuntimeException: Unable to create the cache directory (storage//fd). (uncaught exception) at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Cache/Filesystem.php line 60 {"exception":"[object] (RuntimeException(code: 0): Unable to create the cache directory (storage//fd). at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Cache/Filesystem.php:60)"} []
[2015-12-21 18:40:56] app.CRITICAL: RuntimeException: Unable to create the cache directory (storage/fd). (uncaught exception) at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Cache/Filesystem.php line 60 {"exception":"[object] (RuntimeException(code: 0): Unable to create the cache directory (storage/fd). at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Cache/Filesystem.php:60)"} []
[2015-12-21 18:41:03] app.CRITICAL: RuntimeException: Unable to create the cache directory (storage/cache//fd). (uncaught exception) at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Cache/Filesystem.php line 60 {"exception":"[object] (RuntimeException(code: 0): Unable to create the cache directory (storage/cache//fd). at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Cache/Filesystem.php:60)"} []
[2015-12-21 18:41:07] app.CRITICAL: RuntimeException: Unable to create the cache directory (storage/cache/fd). (uncaught exception) at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Cache/Filesystem.php line 60 {"exception":"[object] (RuntimeException(code: 0): Unable to create the cache directory (storage/cache/fd). at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Cache/Filesystem.php:60)"} []
[2015-12-21 18:41:16] app.CRITICAL: RuntimeException: Unable to create the cache directory (storage/cache/fd). (uncaught exception) at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Cache/Filesystem.php line 60 {"exception":"[object] (RuntimeException(code: 0): Unable to create the cache directory (storage/cache/fd). at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Cache/Filesystem.php:60)"} []
[2015-12-21 18:41:29] app.CRITICAL: RuntimeException: Unable to create the cache directory (/var/www/admin-portal/storage/cache/fd). (uncaught exception) at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Cache/Filesystem.php line 60 {"exception":"[object] (RuntimeException(code: 0): Unable to create the cache directory (/var/www/admin-portal/storage/cache/fd). at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Cache/Filesystem.php:60)"} []
[2015-12-21 18:44:24] app.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException: No route found for "GET /index.html" (from "http://admin.portal.dev/eleitoral") (uncaught exception) at /var/www/admin-portal/vendor/symfony/http-kernel/EventListener/RouterListener.php line 123 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /index.html\" (from \"http://admin.portal.dev/eleitoral\") at /var/www/admin-portal/vendor/symfony/http-kernel/EventListener/RouterListener.php:123, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/index.html\". at /var/www/admin-portal/vendor/symfony/routing/Matcher/UrlMatcher.php:98)"} []
[2015-12-21 18:44:41] app.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException: No route found for "GET /index.html" (from "http://admin.portal.dev/eleitoral") (uncaught exception) at /var/www/admin-portal/vendor/symfony/http-kernel/EventListener/RouterListener.php line 123 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /index.html\" (from \"http://admin.portal.dev/eleitoral\") at /var/www/admin-portal/vendor/symfony/http-kernel/EventListener/RouterListener.php:123, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/index.html\". at /var/www/admin-portal/vendor/symfony/routing/Matcher/UrlMatcher.php:98)"} []
[2015-12-22 11:19:56] app.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException: No route found for "GET /favicon.ico" (uncaught exception) at /var/www/admin-portal/vendor/symfony/http-kernel/EventListener/RouterListener.php line 123 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /favicon.ico\" at /var/www/admin-portal/vendor/symfony/http-kernel/EventListener/RouterListener.php:123, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/favicon.ico\". at /var/www/admin-portal/vendor/symfony/routing/Matcher/UrlMatcher.php:98)"} []
[2015-12-22 11:37:43] app.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException: No route found for "GET /admin" (uncaught exception) at /var/www/admin-portal/vendor/symfony/http-kernel/EventListener/RouterListener.php line 123 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /admin\" at /var/www/admin-portal/vendor/symfony/http-kernel/EventListener/RouterListener.php:123, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/admin\". at /var/www/admin-portal/vendor/symfony/routing/Matcher/UrlMatcher.php:98)"} []
[2015-12-22 11:38:43] app.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException: No route found for "GET /admin" (uncaught exception) at /var/www/admin-portal/vendor/symfony/http-kernel/EventListener/RouterListener.php line 123 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /admin\" at /var/www/admin-portal/vendor/symfony/http-kernel/EventListener/RouterListener.php:123, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/admin\". at /var/www/admin-portal/vendor/symfony/routing/Matcher/UrlMatcher.php:98)"} []
[2015-12-22 11:48:53] app.CRITICAL: InvalidArgumentException: Identifier "session" is not defined. (uncaught exception) at /var/www/admin-portal/vendor/pimple/pimple/lib/Pimple.php line 78 {"exception":"[object] (InvalidArgumentException(code: 0): Identifier \"session\" is not defined. at /var/www/admin-portal/vendor/pimple/pimple/lib/Pimple.php:78)"} []
[2015-12-22 11:49:09] app.CRITICAL: InvalidArgumentException: Identifier "session" is not defined. (uncaught exception) at /var/www/admin-portal/vendor/pimple/pimple/lib/Pimple.php line 78 {"exception":"[object] (InvalidArgumentException(code: 0): Identifier \"session\" is not defined. at /var/www/admin-portal/vendor/pimple/pimple/lib/Pimple.php:78)"} []
[2015-12-22 11:49:20] app.CRITICAL: InvalidArgumentException: Identifier "session" is not defined. (uncaught exception) at /var/www/admin-portal/vendor/pimple/pimple/lib/Pimple.php line 78 {"exception":"[object] (InvalidArgumentException(code: 0): Identifier \"session\" is not defined. at /var/www/admin-portal/vendor/pimple/pimple/lib/Pimple.php:78)"} []
[2015-12-22 11:50:02] app.CRITICAL: InvalidArgumentException: Identifier "session" is not defined. (uncaught exception) at /var/www/admin-portal/vendor/pimple/pimple/lib/Pimple.php line 78 {"exception":"[object] (InvalidArgumentException(code: 0): Identifier \"session\" is not defined. at /var/www/admin-portal/vendor/pimple/pimple/lib/Pimple.php:78)"} []
[2015-12-22 11:50:02] app.CRITICAL: InvalidArgumentException: Identifier "session" is not defined. (uncaught exception) at /var/www/admin-portal/vendor/pimple/pimple/lib/Pimple.php line 78 {"exception":"[object] (InvalidArgumentException(code: 0): Identifier \"session\" is not defined. at /var/www/admin-portal/vendor/pimple/pimple/lib/Pimple.php:78)"} []
[2015-12-22 11:50:03] app.CRITICAL: InvalidArgumentException: Identifier "session" is not defined. (uncaught exception) at /var/www/admin-portal/vendor/pimple/pimple/lib/Pimple.php line 78 {"exception":"[object] (InvalidArgumentException(code: 0): Identifier \"session\" is not defined. at /var/www/admin-portal/vendor/pimple/pimple/lib/Pimple.php:78)"} []
[2015-12-22 11:50:03] app.CRITICAL: InvalidArgumentException: Identifier "session" is not defined. (uncaught exception) at /var/www/admin-portal/vendor/pimple/pimple/lib/Pimple.php line 78 {"exception":"[object] (InvalidArgumentException(code: 0): Identifier \"session\" is not defined. at /var/www/admin-portal/vendor/pimple/pimple/lib/Pimple.php:78)"} []
[2015-12-22 11:50:03] app.CRITICAL: InvalidArgumentException: Identifier "session" is not defined. (uncaught exception) at /var/www/admin-portal/vendor/pimple/pimple/lib/Pimple.php line 78 {"exception":"[object] (InvalidArgumentException(code: 0): Identifier \"session\" is not defined. at /var/www/admin-portal/vendor/pimple/pimple/lib/Pimple.php:78)"} []
[2015-12-22 11:50:03] app.CRITICAL: InvalidArgumentException: Identifier "session" is not defined. (uncaught exception) at /var/www/admin-portal/vendor/pimple/pimple/lib/Pimple.php line 78 {"exception":"[object] (InvalidArgumentException(code: 0): Identifier \"session\" is not defined. at /var/www/admin-portal/vendor/pimple/pimple/lib/Pimple.php:78)"} []
[2015-12-22 11:50:06] app.CRITICAL: InvalidArgumentException: Identifier "session" is not defined. (uncaught exception) at /var/www/admin-portal/vendor/pimple/pimple/lib/Pimple.php line 78 {"exception":"[object] (InvalidArgumentException(code: 0): Identifier \"session\" is not defined. at /var/www/admin-portal/vendor/pimple/pimple/lib/Pimple.php:78)"} []
[2015-12-22 11:59:16] app.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException: No route found for "GET /foo" (uncaught exception) at /var/www/admin-portal/vendor/symfony/http-kernel/EventListener/RouterListener.php line 123 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /foo\" at /var/www/admin-portal/vendor/symfony/http-kernel/EventListener/RouterListener.php:123, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/foo\". at /var/www/admin-portal/vendor/symfony/routing/Matcher/UrlMatcher.php:98)"} []
[2015-12-22 13:41:56] app.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException: No route found for "GET /admin" (uncaught exception) at /var/www/admin-portal/vendor/symfony/http-kernel/EventListener/RouterListener.php line 123 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /admin\" at /var/www/admin-portal/vendor/symfony/http-kernel/EventListener/RouterListener.php:123, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/admin\". at /var/www/admin-portal/vendor/symfony/routing/Matcher/UrlMatcher.php:98)"} []
[2015-12-22 13:46:06] app.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException: No route found for "GET /eleitoral" (uncaught exception) at /var/www/admin-portal/vendor/symfony/http-kernel/EventListener/RouterListener.php line 123 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /eleitoral\" at /var/www/admin-portal/vendor/symfony/http-kernel/EventListener/RouterListener.php:123, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/eleitoral\". at /var/www/admin-portal/vendor/symfony/routing/Matcher/UrlMatcher.php:98)"} []
[2015-12-22 13:48:15] app.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException: No route found for "GET /eleitoral" (from "http://admin.portal.dev/login") (uncaught exception) at /var/www/admin-portal/vendor/symfony/http-kernel/EventListener/RouterListener.php line 123 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /eleitoral\" (from \"http://admin.portal.dev/login\") at /var/www/admin-portal/vendor/symfony/http-kernel/EventListener/RouterListener.php:123, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/eleitoral\". at /var/www/admin-portal/vendor/symfony/routing/Matcher/UrlMatcher.php:98)"} []
[2015-12-22 13:49:06] app.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException: No route found for "GET /eleitoral" (from "http://admin.portal.dev/login") (uncaught exception) at /var/www/admin-portal/vendor/symfony/http-kernel/EventListener/RouterListener.php line 123 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /eleitoral\" (from \"http://admin.portal.dev/login\") at /var/www/admin-portal/vendor/symfony/http-kernel/EventListener/RouterListener.php:123, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/eleitoral\". at /var/www/admin-portal/vendor/symfony/routing/Matcher/UrlMatcher.php:98)"} []
[2015-12-22 13:50:21] app.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException: No route found for "GET /eleitoral" (from "http://admin.portal.dev/login") (uncaught exception) at /var/www/admin-portal/vendor/symfony/http-kernel/EventListener/RouterListener.php line 123 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /eleitoral\" (from \"http://admin.portal.dev/login\") at /var/www/admin-portal/vendor/symfony/http-kernel/EventListener/RouterListener.php:123, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/eleitoral\". at /var/www/admin-portal/vendor/symfony/routing/Matcher/UrlMatcher.php:98)"} []
[2015-12-22 14:00:29] app.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException: No route found for "GET /eleitoral" (from "http://admin.portal.dev/login") (uncaught exception) at /var/www/admin-portal/vendor/symfony/http-kernel/EventListener/RouterListener.php line 123 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /eleitoral\" (from \"http://admin.portal.dev/login\") at /var/www/admin-portal/vendor/symfony/http-kernel/EventListener/RouterListener.php:123, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/eleitoral\". at /var/www/admin-portal/vendor/symfony/routing/Matcher/UrlMatcher.php:98)"} []
[2015-12-22 15:45:47] app.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException: No route found for "GET /eleitoral" (from "http://admin.portal.dev/login") (uncaught exception) at /var/www/admin-portal/vendor/symfony/http-kernel/EventListener/RouterListener.php line 123 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /eleitoral\" (from \"http://admin.portal.dev/login\") at /var/www/admin-portal/vendor/symfony/http-kernel/EventListener/RouterListener.php:123, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/eleitoral\". at /var/www/admin-portal/vendor/symfony/routing/Matcher/UrlMatcher.php:98)"} []
[2015-12-22 15:46:53] app.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException: No route found for "GET /eleitoral" (from "http://admin.portal.dev/login") (uncaught exception) at /var/www/admin-portal/vendor/symfony/http-kernel/EventListener/RouterListener.php line 123 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /eleitoral\" (from \"http://admin.portal.dev/login\") at /var/www/admin-portal/vendor/symfony/http-kernel/EventListener/RouterListener.php:123, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/eleitoral\". at /var/www/admin-portal/vendor/symfony/routing/Matcher/UrlMatcher.php:98)"} []
[2015-12-22 15:50:25] app.WARNING: Unable to look for the controller as the "_controller" parameter is missing. [] []
[2015-12-22 15:50:25] app.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException: Unable to find the controller for path "/login_check". The route is wrongly configured. (uncaught exception) at /var/www/admin-portal/vendor/symfony/http-kernel/HttpKernel.php line 128 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): Unable to find the controller for path \"/login_check\". The route is wrongly configured. at /var/www/admin-portal/vendor/symfony/http-kernel/HttpKernel.php:128)"} []
[2015-12-22 15:51:05] app.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException: No route found for "GET /dsadsa" (uncaught exception) at /var/www/admin-portal/vendor/symfony/http-kernel/EventListener/RouterListener.php line 123 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /dsadsa\" at /var/www/admin-portal/vendor/symfony/http-kernel/EventListener/RouterListener.php:123, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/dsadsa\". at /var/www/admin-portal/vendor/symfony/routing/Matcher/UrlMatcher.php:98)"} []
[2015-12-22 15:58:36] app.CRITICAL: Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("Unable to generate a URL for the named route "admin/login_check" as such route does not exist.") in "forms/layouts/layout-login.tpl" at line 43. (uncaught exception) at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php line 396 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Unable to generate a URL for the named route \"admin/login_check\" as such route does not exist.\") in \"forms/layouts/layout-login.tpl\" at line 43. at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php:396, Symfony\\Component\\Routing\\Exception\\RouteNotFoundException(code: 0): Unable to generate a URL for the named route \"admin/login_check\" as such route does not exist. at /var/www/admin-portal/vendor/symfony/routing/Generator/UrlGenerator.php:130)"} []
[2015-12-22 15:58:57] app.CRITICAL: Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("Unable to generate a URL for the named route "login_check" as such route does not exist.") in "forms/layouts/layout-login.tpl" at line 43. (uncaught exception) at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php line 396 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Unable to generate a URL for the named route \"login_check\" as such route does not exist.\") in \"forms/layouts/layout-login.tpl\" at line 43. at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php:396, Symfony\\Component\\Routing\\Exception\\RouteNotFoundException(code: 0): Unable to generate a URL for the named route \"login_check\" as such route does not exist. at /var/www/admin-portal/vendor/symfony/routing/Generator/UrlGenerator.php:130)"} []
[2015-12-22 15:59:09] app.CRITICAL: Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("Unable to generate a URL for the named route "/admin/login_check" as such route does not exist.") in "forms/layouts/layout-login.tpl" at line 43. (uncaught exception) at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php line 396 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Unable to generate a URL for the named route \"/admin/login_check\" as such route does not exist.\") in \"forms/layouts/layout-login.tpl\" at line 43. at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php:396, Symfony\\Component\\Routing\\Exception\\RouteNotFoundException(code: 0): Unable to generate a URL for the named route \"/admin/login_check\" as such route does not exist. at /var/www/admin-portal/vendor/symfony/routing/Generator/UrlGenerator.php:130)"} []
[2015-12-22 16:07:18] app.CRITICAL: Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("Unable to generate a URL for the named route "/admin/login_check" as such route does not exist.") in "forms/layouts/layout-login.tpl" at line 43. (uncaught exception) at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php line 396 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Unable to generate a URL for the named route \"/admin/login_check\" as such route does not exist.\") in \"forms/layouts/layout-login.tpl\" at line 43. at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php:396, Symfony\\Component\\Routing\\Exception\\RouteNotFoundException(code: 0): Unable to generate a URL for the named route \"/admin/login_check\" as such route does not exist. at /var/www/admin-portal/vendor/symfony/routing/Generator/UrlGenerator.php:130)"} []
[2015-12-22 16:07:31] app.CRITICAL: Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("Unable to generate a URL for the named route "admin/login_check" as such route does not exist.") in "forms/layouts/layout-login.tpl" at line 43. (uncaught exception) at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php line 396 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Unable to generate a URL for the named route \"admin/login_check\" as such route does not exist.\") in \"forms/layouts/layout-login.tpl\" at line 43. at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php:396, Symfony\\Component\\Routing\\Exception\\RouteNotFoundException(code: 0): Unable to generate a URL for the named route \"admin/login_check\" as such route does not exist. at /var/www/admin-portal/vendor/symfony/routing/Generator/UrlGenerator.php:130)"} []
[2015-12-22 16:07:31] app.CRITICAL: Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("Unable to generate a URL for the named route "admin/login_check" as such route does not exist.") in "forms/layouts/layout-login.tpl" at line 43. (uncaught exception) at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php line 396 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Unable to generate a URL for the named route \"admin/login_check\" as such route does not exist.\") in \"forms/layouts/layout-login.tpl\" at line 43. at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php:396, Symfony\\Component\\Routing\\Exception\\RouteNotFoundException(code: 0): Unable to generate a URL for the named route \"admin/login_check\" as such route does not exist. at /var/www/admin-portal/vendor/symfony/routing/Generator/UrlGenerator.php:130)"} []
[2015-12-22 16:07:32] app.CRITICAL: Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("Unable to generate a URL for the named route "admin/login_check" as such route does not exist.") in "forms/layouts/layout-login.tpl" at line 43. (uncaught exception) at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php line 396 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Unable to generate a URL for the named route \"admin/login_check\" as such route does not exist.\") in \"forms/layouts/layout-login.tpl\" at line 43. at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php:396, Symfony\\Component\\Routing\\Exception\\RouteNotFoundException(code: 0): Unable to generate a URL for the named route \"admin/login_check\" as such route does not exist. at /var/www/admin-portal/vendor/symfony/routing/Generator/UrlGenerator.php:130)"} []
[2015-12-22 16:07:35] app.CRITICAL: Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("Unable to generate a URL for the named route "login_check" as such route does not exist.") in "forms/layouts/layout-login.tpl" at line 43. (uncaught exception) at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php line 396 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Unable to generate a URL for the named route \"login_check\" as such route does not exist.\") in \"forms/layouts/layout-login.tpl\" at line 43. at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php:396, Symfony\\Component\\Routing\\Exception\\RouteNotFoundException(code: 0): Unable to generate a URL for the named route \"login_check\" as such route does not exist. at /var/www/admin-portal/vendor/symfony/routing/Generator/UrlGenerator.php:130)"} []
[2015-12-22 16:07:48] app.CRITICAL: Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("Unable to generate a URL for the named route "login_check" as such route does not exist.") in "forms/layouts/layout-login.tpl" at line 43. (uncaught exception) at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php line 396 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Unable to generate a URL for the named route \"login_check\" as such route does not exist.\") in \"forms/layouts/layout-login.tpl\" at line 43. at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php:396, Symfony\\Component\\Routing\\Exception\\RouteNotFoundException(code: 0): Unable to generate a URL for the named route \"login_check\" as such route does not exist. at /var/www/admin-portal/vendor/symfony/routing/Generator/UrlGenerator.php:130)"} []
[2015-12-22 16:07:48] app.CRITICAL: Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("Unable to generate a URL for the named route "login_check" as such route does not exist.") in "forms/layouts/layout-login.tpl" at line 43. (uncaught exception) at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php line 396 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Unable to generate a URL for the named route \"login_check\" as such route does not exist.\") in \"forms/layouts/layout-login.tpl\" at line 43. at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php:396, Symfony\\Component\\Routing\\Exception\\RouteNotFoundException(code: 0): Unable to generate a URL for the named route \"login_check\" as such route does not exist. at /var/www/admin-portal/vendor/symfony/routing/Generator/UrlGenerator.php:130)"} []
[2015-12-22 16:07:49] app.CRITICAL: Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("Unable to generate a URL for the named route "login_check" as such route does not exist.") in "forms/layouts/layout-login.tpl" at line 43. (uncaught exception) at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php line 396 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Unable to generate a URL for the named route \"login_check\" as such route does not exist.\") in \"forms/layouts/layout-login.tpl\" at line 43. at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php:396, Symfony\\Component\\Routing\\Exception\\RouteNotFoundException(code: 0): Unable to generate a URL for the named route \"login_check\" as such route does not exist. at /var/www/admin-portal/vendor/symfony/routing/Generator/UrlGenerator.php:130)"} []
[2015-12-22 16:07:49] app.CRITICAL: Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("Unable to generate a URL for the named route "login_check" as such route does not exist.") in "forms/layouts/layout-login.tpl" at line 43. (uncaught exception) at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php line 396 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Unable to generate a URL for the named route \"login_check\" as such route does not exist.\") in \"forms/layouts/layout-login.tpl\" at line 43. at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php:396, Symfony\\Component\\Routing\\Exception\\RouteNotFoundException(code: 0): Unable to generate a URL for the named route \"login_check\" as such route does not exist. at /var/www/admin-portal/vendor/symfony/routing/Generator/UrlGenerator.php:130)"} []
[2015-12-22 16:07:49] app.CRITICAL: Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("Unable to generate a URL for the named route "login_check" as such route does not exist.") in "forms/layouts/layout-login.tpl" at line 43. (uncaught exception) at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php line 396 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Unable to generate a URL for the named route \"login_check\" as such route does not exist.\") in \"forms/layouts/layout-login.tpl\" at line 43. at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php:396, Symfony\\Component\\Routing\\Exception\\RouteNotFoundException(code: 0): Unable to generate a URL for the named route \"login_check\" as such route does not exist. at /var/www/admin-portal/vendor/symfony/routing/Generator/UrlGenerator.php:130)"} []
[2015-12-22 16:07:50] app.CRITICAL: Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("Unable to generate a URL for the named route "login_check" as such route does not exist.") in "forms/layouts/layout-login.tpl" at line 43. (uncaught exception) at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php line 396 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Unable to generate a URL for the named route \"login_check\" as such route does not exist.\") in \"forms/layouts/layout-login.tpl\" at line 43. at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php:396, Symfony\\Component\\Routing\\Exception\\RouteNotFoundException(code: 0): Unable to generate a URL for the named route \"login_check\" as such route does not exist. at /var/www/admin-portal/vendor/symfony/routing/Generator/UrlGenerator.php:130)"} []
[2015-12-22 16:07:58] app.CRITICAL: Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("Unable to generate a URL for the named route "login_check" as such route does not exist.") in "forms/layouts/layout-login.tpl" at line 43. (uncaught exception) at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php line 396 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Unable to generate a URL for the named route \"login_check\" as such route does not exist.\") in \"forms/layouts/layout-login.tpl\" at line 43. at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php:396, Symfony\\Component\\Routing\\Exception\\RouteNotFoundException(code: 0): Unable to generate a URL for the named route \"login_check\" as such route does not exist. at /var/www/admin-portal/vendor/symfony/routing/Generator/UrlGenerator.php:130)"} []
[2015-12-22 16:07:58] app.CRITICAL: Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("Unable to generate a URL for the named route "login_check" as such route does not exist.") in "forms/layouts/layout-login.tpl" at line 43. (uncaught exception) at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php line 396 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Unable to generate a URL for the named route \"login_check\" as such route does not exist.\") in \"forms/layouts/layout-login.tpl\" at line 43. at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php:396, Symfony\\Component\\Routing\\Exception\\RouteNotFoundException(code: 0): Unable to generate a URL for the named route \"login_check\" as such route does not exist. at /var/www/admin-portal/vendor/symfony/routing/Generator/UrlGenerator.php:130)"} []
[2015-12-22 16:07:58] app.CRITICAL: Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("Unable to generate a URL for the named route "login_check" as such route does not exist.") in "forms/layouts/layout-login.tpl" at line 43. (uncaught exception) at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php line 396 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Unable to generate a URL for the named route \"login_check\" as such route does not exist.\") in \"forms/layouts/layout-login.tpl\" at line 43. at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php:396, Symfony\\Component\\Routing\\Exception\\RouteNotFoundException(code: 0): Unable to generate a URL for the named route \"login_check\" as such route does not exist. at /var/www/admin-portal/vendor/symfony/routing/Generator/UrlGenerator.php:130)"} []
[2015-12-22 16:07:59] app.CRITICAL: Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("Unable to generate a URL for the named route "login_check" as such route does not exist.") in "forms/layouts/layout-login.tpl" at line 43. (uncaught exception) at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php line 396 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Unable to generate a URL for the named route \"login_check\" as such route does not exist.\") in \"forms/layouts/layout-login.tpl\" at line 43. at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php:396, Symfony\\Component\\Routing\\Exception\\RouteNotFoundException(code: 0): Unable to generate a URL for the named route \"login_check\" as such route does not exist. at /var/www/admin-portal/vendor/symfony/routing/Generator/UrlGenerator.php:130)"} []
[2015-12-22 16:07:59] app.CRITICAL: Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("Unable to generate a URL for the named route "login_check" as such route does not exist.") in "forms/layouts/layout-login.tpl" at line 43. (uncaught exception) at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php line 396 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Unable to generate a URL for the named route \"login_check\" as such route does not exist.\") in \"forms/layouts/layout-login.tpl\" at line 43. at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php:396, Symfony\\Component\\Routing\\Exception\\RouteNotFoundException(code: 0): Unable to generate a URL for the named route \"login_check\" as such route does not exist. at /var/www/admin-portal/vendor/symfony/routing/Generator/UrlGenerator.php:130)"} []
[2015-12-22 16:07:59] app.CRITICAL: Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("Unable to generate a URL for the named route "login_check" as such route does not exist.") in "forms/layouts/layout-login.tpl" at line 43. (uncaught exception) at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php line 396 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Unable to generate a URL for the named route \"login_check\" as such route does not exist.\") in \"forms/layouts/layout-login.tpl\" at line 43. at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php:396, Symfony\\Component\\Routing\\Exception\\RouteNotFoundException(code: 0): Unable to generate a URL for the named route \"login_check\" as such route does not exist. at /var/www/admin-portal/vendor/symfony/routing/Generator/UrlGenerator.php:130)"} []
[2015-12-22 16:07:59] app.CRITICAL: Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("Unable to generate a URL for the named route "login_check" as such route does not exist.") in "forms/layouts/layout-login.tpl" at line 43. (uncaught exception) at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php line 396 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Unable to generate a URL for the named route \"login_check\" as such route does not exist.\") in \"forms/layouts/layout-login.tpl\" at line 43. at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php:396, Symfony\\Component\\Routing\\Exception\\RouteNotFoundException(code: 0): Unable to generate a URL for the named route \"login_check\" as such route does not exist. at /var/www/admin-portal/vendor/symfony/routing/Generator/UrlGenerator.php:130)"} []
[2015-12-22 16:07:59] app.CRITICAL: Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("Unable to generate a URL for the named route "login_check" as such route does not exist.") in "forms/layouts/layout-login.tpl" at line 43. (uncaught exception) at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php line 396 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Unable to generate a URL for the named route \"login_check\" as such route does not exist.\") in \"forms/layouts/layout-login.tpl\" at line 43. at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php:396, Symfony\\Component\\Routing\\Exception\\RouteNotFoundException(code: 0): Unable to generate a URL for the named route \"login_check\" as such route does not exist. at /var/www/admin-portal/vendor/symfony/routing/Generator/UrlGenerator.php:130)"} []
[2015-12-22 16:08:02] app.CRITICAL: Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("Unable to generate a URL for the named route "login_check" as such route does not exist.") in "forms/layouts/layout-login.tpl" at line 43. (uncaught exception) at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php line 396 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Unable to generate a URL for the named route \"login_check\" as such route does not exist.\") in \"forms/layouts/layout-login.tpl\" at line 43. at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php:396, Symfony\\Component\\Routing\\Exception\\RouteNotFoundException(code: 0): Unable to generate a URL for the named route \"login_check\" as such route does not exist. at /var/www/admin-portal/vendor/symfony/routing/Generator/UrlGenerator.php:130)"} []
[2015-12-22 16:08:46] app.CRITICAL: Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("Unable to generate a URL for the named route "login_check" as such route does not exist.") in "forms/layouts/layout-login.tpl" at line 43. (uncaught exception) at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php line 396 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Unable to generate a URL for the named route \"login_check\" as such route does not exist.\") in \"forms/layouts/layout-login.tpl\" at line 43. at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php:396, Symfony\\Component\\Routing\\Exception\\RouteNotFoundException(code: 0): Unable to generate a URL for the named route \"login_check\" as such route does not exist. at /var/www/admin-portal/vendor/symfony/routing/Generator/UrlGenerator.php:130)"} []
[2015-12-22 16:08:46] app.CRITICAL: Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("Unable to generate a URL for the named route "login_check" as such route does not exist.") in "forms/layouts/layout-login.tpl" at line 43. (uncaught exception) at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php line 396 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Unable to generate a URL for the named route \"login_check\" as such route does not exist.\") in \"forms/layouts/layout-login.tpl\" at line 43. at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php:396, Symfony\\Component\\Routing\\Exception\\RouteNotFoundException(code: 0): Unable to generate a URL for the named route \"login_check\" as such route does not exist. at /var/www/admin-portal/vendor/symfony/routing/Generator/UrlGenerator.php:130)"} []
[2015-12-22 16:08:47] app.CRITICAL: Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("Unable to generate a URL for the named route "login_check" as such route does not exist.") in "forms/layouts/layout-login.tpl" at line 43. (uncaught exception) at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php line 396 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Unable to generate a URL for the named route \"login_check\" as such route does not exist.\") in \"forms/layouts/layout-login.tpl\" at line 43. at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php:396, Symfony\\Component\\Routing\\Exception\\RouteNotFoundException(code: 0): Unable to generate a URL for the named route \"login_check\" as such route does not exist. at /var/www/admin-portal/vendor/symfony/routing/Generator/UrlGenerator.php:130)"} []
[2015-12-22 16:08:55] app.CRITICAL: Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("Unable to generate a URL for the named route "login_check" as such route does not exist.") in "forms/layouts/layout-login.tpl" at line 43. (uncaught exception) at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php line 396 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Unable to generate a URL for the named route \"login_check\" as such route does not exist.\") in \"forms/layouts/layout-login.tpl\" at line 43. at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php:396, Symfony\\Component\\Routing\\Exception\\RouteNotFoundException(code: 0): Unable to generate a URL for the named route \"login_check\" as such route does not exist. at /var/www/admin-portal/vendor/symfony/routing/Generator/UrlGenerator.php:130)"} []
[2015-12-22 16:09:04] app.CRITICAL: Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("Unable to generate a URL for the named route "login_check" as such route does not exist.") in "forms/layouts/layout-login.tpl" at line 43. (uncaught exception) at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php line 396 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Unable to generate a URL for the named route \"login_check\" as such route does not exist.\") in \"forms/layouts/layout-login.tpl\" at line 43. at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php:396, Symfony\\Component\\Routing\\Exception\\RouteNotFoundException(code: 0): Unable to generate a URL for the named route \"login_check\" as such route does not exist. at /var/www/admin-portal/vendor/symfony/routing/Generator/UrlGenerator.php:130)"} []
[2015-12-22 16:09:04] app.CRITICAL: Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("Unable to generate a URL for the named route "login_check" as such route does not exist.") in "forms/layouts/layout-login.tpl" at line 43. (uncaught exception) at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php line 396 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Unable to generate a URL for the named route \"login_check\" as such route does not exist.\") in \"forms/layouts/layout-login.tpl\" at line 43. at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php:396, Symfony\\Component\\Routing\\Exception\\RouteNotFoundException(code: 0): Unable to generate a URL for the named route \"login_check\" as such route does not exist. at /var/www/admin-portal/vendor/symfony/routing/Generator/UrlGenerator.php:130)"} []
[2015-12-22 16:09:04] app.CRITICAL: Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("Unable to generate a URL for the named route "login_check" as such route does not exist.") in "forms/layouts/layout-login.tpl" at line 43. (uncaught exception) at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php line 396 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Unable to generate a URL for the named route \"login_check\" as such route does not exist.\") in \"forms/layouts/layout-login.tpl\" at line 43. at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php:396, Symfony\\Component\\Routing\\Exception\\RouteNotFoundException(code: 0): Unable to generate a URL for the named route \"login_check\" as such route does not exist. at /var/www/admin-portal/vendor/symfony/routing/Generator/UrlGenerator.php:130)"} []
[2015-12-22 16:09:04] app.CRITICAL: Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("Unable to generate a URL for the named route "login_check" as such route does not exist.") in "forms/layouts/layout-login.tpl" at line 43. (uncaught exception) at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php line 396 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Unable to generate a URL for the named route \"login_check\" as such route does not exist.\") in \"forms/layouts/layout-login.tpl\" at line 43. at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php:396, Symfony\\Component\\Routing\\Exception\\RouteNotFoundException(code: 0): Unable to generate a URL for the named route \"login_check\" as such route does not exist. at /var/www/admin-portal/vendor/symfony/routing/Generator/UrlGenerator.php:130)"} []
[2015-12-22 16:09:05] app.CRITICAL: Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("Unable to generate a URL for the named route "login_check" as such route does not exist.") in "forms/layouts/layout-login.tpl" at line 43. (uncaught exception) at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php line 396 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Unable to generate a URL for the named route \"login_check\" as such route does not exist.\") in \"forms/layouts/layout-login.tpl\" at line 43. at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php:396, Symfony\\Component\\Routing\\Exception\\RouteNotFoundException(code: 0): Unable to generate a URL for the named route \"login_check\" as such route does not exist. at /var/www/admin-portal/vendor/symfony/routing/Generator/UrlGenerator.php:130)"} []
[2015-12-22 16:09:09] app.CRITICAL: Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("Unable to generate a URL for the named route "login_check" as such route does not exist.") in "forms/layouts/layout-login.tpl" at line 43. (uncaught exception) at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php line 396 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Unable to generate a URL for the named route \"login_check\" as such route does not exist.\") in \"forms/layouts/layout-login.tpl\" at line 43. at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php:396, Symfony\\Component\\Routing\\Exception\\RouteNotFoundException(code: 0): Unable to generate a URL for the named route \"login_check\" as such route does not exist. at /var/www/admin-portal/vendor/symfony/routing/Generator/UrlGenerator.php:130)"} []
[2015-12-22 16:09:09] app.CRITICAL: Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("Unable to generate a URL for the named route "login_check" as such route does not exist.") in "forms/layouts/layout-login.tpl" at line 43. (uncaught exception) at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php line 396 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Unable to generate a URL for the named route \"login_check\" as such route does not exist.\") in \"forms/layouts/layout-login.tpl\" at line 43. at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php:396, Symfony\\Component\\Routing\\Exception\\RouteNotFoundException(code: 0): Unable to generate a URL for the named route \"login_check\" as such route does not exist. at /var/www/admin-portal/vendor/symfony/routing/Generator/UrlGenerator.php:130)"} []
[2015-12-22 16:09:29] app.CRITICAL: Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("Unable to generate a URL for the named route "login_check" as such route does not exist.") in "forms/layouts/layout-login.tpl" at line 43. (uncaught exception) at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php line 396 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Unable to generate a URL for the named route \"login_check\" as such route does not exist.\") in \"forms/layouts/layout-login.tpl\" at line 43. at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php:396, Symfony\\Component\\Routing\\Exception\\RouteNotFoundException(code: 0): Unable to generate a URL for the named route \"login_check\" as such route does not exist. at /var/www/admin-portal/vendor/symfony/routing/Generator/UrlGenerator.php:130)"} []
[2015-12-22 16:09:29] app.CRITICAL: Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("Unable to generate a URL for the named route "login_check" as such route does not exist.") in "forms/layouts/layout-login.tpl" at line 43. (uncaught exception) at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php line 396 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Unable to generate a URL for the named route \"login_check\" as such route does not exist.\") in \"forms/layouts/layout-login.tpl\" at line 43. at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php:396, Symfony\\Component\\Routing\\Exception\\RouteNotFoundException(code: 0): Unable to generate a URL for the named route \"login_check\" as such route does not exist. at /var/www/admin-portal/vendor/symfony/routing/Generator/UrlGenerator.php:130)"} []
[2015-12-22 16:09:30] app.CRITICAL: Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("Unable to generate a URL for the named route "login_check" as such route does not exist.") in "forms/layouts/layout-login.tpl" at line 43. (uncaught exception) at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php line 396 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Unable to generate a URL for the named route \"login_check\" as such route does not exist.\") in \"forms/layouts/layout-login.tpl\" at line 43. at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php:396, Symfony\\Component\\Routing\\Exception\\RouteNotFoundException(code: 0): Unable to generate a URL for the named route \"login_check\" as such route does not exist. at /var/www/admin-portal/vendor/symfony/routing/Generator/UrlGenerator.php:130)"} []
[2015-12-22 16:09:30] app.CRITICAL: Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("Unable to generate a URL for the named route "login_check" as such route does not exist.") in "forms/layouts/layout-login.tpl" at line 43. (uncaught exception) at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php line 396 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Unable to generate a URL for the named route \"login_check\" as such route does not exist.\") in \"forms/layouts/layout-login.tpl\" at line 43. at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php:396, Symfony\\Component\\Routing\\Exception\\RouteNotFoundException(code: 0): Unable to generate a URL for the named route \"login_check\" as such route does not exist. at /var/www/admin-portal/vendor/symfony/routing/Generator/UrlGenerator.php:130)"} []
[2015-12-22 16:09:30] app.CRITICAL: Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("Unable to generate a URL for the named route "login_check" as such route does not exist.") in "forms/layouts/layout-login.tpl" at line 43. (uncaught exception) at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php line 396 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Unable to generate a URL for the named route \"login_check\" as such route does not exist.\") in \"forms/layouts/layout-login.tpl\" at line 43. at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php:396, Symfony\\Component\\Routing\\Exception\\RouteNotFoundException(code: 0): Unable to generate a URL for the named route \"login_check\" as such route does not exist. at /var/www/admin-portal/vendor/symfony/routing/Generator/UrlGenerator.php:130)"} []
[2015-12-22 16:09:30] app.CRITICAL: Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("Unable to generate a URL for the named route "login_check" as such route does not exist.") in "forms/layouts/layout-login.tpl" at line 43. (uncaught exception) at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php line 396 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Unable to generate a URL for the named route \"login_check\" as such route does not exist.\") in \"forms/layouts/layout-login.tpl\" at line 43. at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php:396, Symfony\\Component\\Routing\\Exception\\RouteNotFoundException(code: 0): Unable to generate a URL for the named route \"login_check\" as such route does not exist. at /var/www/admin-portal/vendor/symfony/routing/Generator/UrlGenerator.php:130)"} []
[2015-12-22 16:09:30] app.CRITICAL: Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("Unable to generate a URL for the named route "login_check" as such route does not exist.") in "forms/layouts/layout-login.tpl" at line 43. (uncaught exception) at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php line 396 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Unable to generate a URL for the named route \"login_check\" as such route does not exist.\") in \"forms/layouts/layout-login.tpl\" at line 43. at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php:396, Symfony\\Component\\Routing\\Exception\\RouteNotFoundException(code: 0): Unable to generate a URL for the named route \"login_check\" as such route does not exist. at /var/www/admin-portal/vendor/symfony/routing/Generator/UrlGenerator.php:130)"} []
[2015-12-22 16:09:30] app.CRITICAL: Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("Unable to generate a URL for the named route "login_check" as such route does not exist.") in "forms/layouts/layout-login.tpl" at line 43. (uncaught exception) at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php line 396 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Unable to generate a URL for the named route \"login_check\" as such route does not exist.\") in \"forms/layouts/layout-login.tpl\" at line 43. at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php:396, Symfony\\Component\\Routing\\Exception\\RouteNotFoundException(code: 0): Unable to generate a URL for the named route \"login_check\" as such route does not exist. at /var/www/admin-portal/vendor/symfony/routing/Generator/UrlGenerator.php:130)"} []
[2015-12-22 16:09:30] app.CRITICAL: Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("Unable to generate a URL for the named route "login_check" as such route does not exist.") in "forms/layouts/layout-login.tpl" at line 43. (uncaught exception) at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php line 396 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Unable to generate a URL for the named route \"login_check\" as such route does not exist.\") in \"forms/layouts/layout-login.tpl\" at line 43. at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php:396, Symfony\\Component\\Routing\\Exception\\RouteNotFoundException(code: 0): Unable to generate a URL for the named route \"login_check\" as such route does not exist. at /var/www/admin-portal/vendor/symfony/routing/Generator/UrlGenerator.php:130)"} []
[2015-12-22 16:09:46] app.CRITICAL: Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("Unable to generate a URL for the named route "login_check" as such route does not exist.") in "forms/layouts/layout-login.tpl" at line 43. (uncaught exception) at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php line 396 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Unable to generate a URL for the named route \"login_check\" as such route does not exist.\") in \"forms/layouts/layout-login.tpl\" at line 43. at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php:396, Symfony\\Component\\Routing\\Exception\\RouteNotFoundException(code: 0): Unable to generate a URL for the named route \"login_check\" as such route does not exist. at /var/www/admin-portal/vendor/symfony/routing/Generator/UrlGenerator.php:130)"} []
[2015-12-22 16:17:56] app.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException: No route found for "GET /sistema/" (uncaught exception) at /var/www/admin-portal/vendor/symfony/http-kernel/EventListener/RouterListener.php line 123 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /sistema/\" at /var/www/admin-portal/vendor/symfony/http-kernel/EventListener/RouterListener.php:123, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/sistema/\". at /var/www/admin-portal/vendor/symfony/routing/Matcher/UrlMatcher.php:98)"} []
[2015-12-22 16:29:43] app.CRITICAL: Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("Unable to generate a URL for the named route "admin_login_check" as such route does not exist.") in "forms/layouts/layout-login.tpl" at line 43. (uncaught exception) at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php line 396 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Unable to generate a URL for the named route \"admin_login_check\" as such route does not exist.\") in \"forms/layouts/layout-login.tpl\" at line 43. at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php:396, Symfony\\Component\\Routing\\Exception\\RouteNotFoundException(code: 0): Unable to generate a URL for the named route \"admin_login_check\" as such route does not exist. at /var/www/admin-portal/vendor/symfony/routing/Generator/UrlGenerator.php:130)"} []
[2015-12-22 17:06:30] app.WARNING: Username could not be found in the selected user provider. {"username":null,"provider":"app\\Models\\Usuarios\\usuarios"} []
[2015-12-22 17:08:34] app.WARNING: Username could not be found in the selected user provider. {"username":null,"provider":"app\\Models\\Usuarios\\usuarios"} []
[2015-12-22 17:13:23] app.WARNING: Username could not be found in the selected user provider. {"username":null,"provider":"app\\Models\\Usuarios\\usuarios"} []
[2015-12-22 17:13:28] app.WARNING: Username could not be found in the selected user provider. {"username":null,"provider":"app\\Models\\Usuarios\\usuarios"} []
[2015-12-22 17:14:08] app.WARNING: Username could not be found in the selected user provider. {"username":null,"provider":"app\\Models\\Usuarios\\usuarios"} []
[2015-12-22 17:14:09] app.WARNING: Username could not be found in the selected user provider. {"username":null,"provider":"app\\Models\\Usuarios\\usuarios"} []
[2015-12-22 17:24:51] app.WARNING: Username could not be found in the selected user provider. {"username":null,"provider":"app\\Models\\Usuarios\\usuarios"} []
[2015-12-22 17:24:54] app.WARNING: Username could not be found in the selected user provider. {"username":null,"provider":"app\\Models\\Usuarios\\usuarios"} []
[2015-12-22 17:24:56] app.WARNING: Username could not be found in the selected user provider. {"username":null,"provider":"app\\Models\\Usuarios\\usuarios"} []
[2015-12-22 17:25:18] app.WARNING: Username could not be found in the selected user provider. {"username":null,"provider":"app\\Models\\Usuarios\\usuarios"} []
[2015-12-22 17:27:35] app.WARNING: Username could not be found in the selected user provider. {"username":null,"provider":"app\\Models\\Usuarios\\usuarios"} []
[2015-12-22 17:27:37] app.WARNING: Username could not be found in the selected user provider. {"username":null,"provider":"app\\Models\\Usuarios\\usuarios"} []
[2015-12-22 17:31:10] app.WARNING: Username could not be found in the selected user provider. {"username":null,"provider":"app\\Models\\Usuarios\\usuarios"} []
[2015-12-22 17:31:11] app.WARNING: Username could not be found in the selected user provider. {"username":null,"provider":"app\\Models\\Usuarios\\usuarios"} []
[2015-12-22 17:31:30] app.WARNING: Username could not be found in the selected user provider. {"username":null,"provider":"app\\Models\\Usuarios\\usuarios"} []
[2015-12-22 17:31:32] app.WARNING: Username could not be found in the selected user provider. {"username":null,"provider":"app\\Models\\Usuarios\\usuarios"} []
[2015-12-22 17:34:38] app.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException: No route found for "GET /sistema/" (uncaught exception) at /var/www/admin-portal/vendor/symfony/http-kernel/EventListener/RouterListener.php line 123 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /sistema/\" at /var/www/admin-portal/vendor/symfony/http-kernel/EventListener/RouterListener.php:123, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/sistema/\". at /var/www/admin-portal/vendor/symfony/routing/Matcher/UrlMatcher.php:98)"} []
[2015-12-22 17:34:45] app.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException: No route found for "GET /sistema/" (uncaught exception) at /var/www/admin-portal/vendor/symfony/http-kernel/EventListener/RouterListener.php line 123 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /sistema/\" at /var/www/admin-portal/vendor/symfony/http-kernel/EventListener/RouterListener.php:123, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/sistema/\". at /var/www/admin-portal/vendor/symfony/routing/Matcher/UrlMatcher.php:98)"} []
[2015-12-22 17:34:56] app.WARNING: Username could not be found in the selected user provider. {"username":null,"provider":"app\\Models\\Usuarios\\usuarios"} []
[2015-12-22 17:37:28] app.WARNING: Username could not be found in the selected user provider. {"username":null,"provider":"app\\Models\\Usuarios\\usuarios"} []
[2015-12-22 17:37:30] app.WARNING: Username could not be found in the selected user provider. {"username":null,"provider":"app\\Models\\Usuarios\\usuarios"} []
[2015-12-22 17:38:17] app.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException: No route found for "POST /sistema/login_check" (from "http://admin.portal.dev/login") (uncaught exception) at /var/www/admin-portal/vendor/symfony/http-kernel/EventListener/RouterListener.php line 123 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"POST /sistema/login_check\" (from \"http://admin.portal.dev/login\") at /var/www/admin-portal/vendor/symfony/http-kernel/EventListener/RouterListener.php:123, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/sistema/login_check\". at /var/www/admin-portal/vendor/symfony/routing/Matcher/UrlMatcher.php:98)"} []
[2015-12-22 17:45:35] app.CRITICAL: Twig_Error_Runtime: An exception has been thrown during the rendering of a template ("Unable to generate a URL for the named route "sistema_login_check" as such route does not exist.") in "forms/layouts/layout-login.tpl" at line 43. (uncaught exception) at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php line 396 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Unable to generate a URL for the named route \"sistema_login_check\" as such route does not exist.\") in \"forms/layouts/layout-login.tpl\" at line 43. at /var/www/admin-portal/vendor/twig/twig/lib/Twig/Template.php:396, Symfony\\Component\\Routing\\Exception\\RouteNotFoundException(code: 0): Unable to generate a URL for the named route \"sistema_login_check\" as such route does not exist. at /var/www/admin-portal/vendor/symfony/routing/Generator/UrlGenerator.php:130)"} []
[2015-12-22 17:46:01] app.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException: No route found for "GET /sistema/login_check" (from "http://admin.portal.dev/login") (uncaught exception) at /var/www/admin-portal/vendor/symfony/http-kernel/EventListener/RouterListener.php line 123 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /sistema/login_check\" (from \"http://admin.portal.dev/login\") at /var/www/admin-portal/vendor/symfony/http-kernel/EventListener/RouterListener.php:123, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/sistema/login_check\". at /var/www/admin-portal/vendor/symfony/routing/Matcher/UrlMatcher.php:98)"} []
[2015-12-22 17:46:10] app.WARNING: Username could not be found in the selected user provider. {"username":null,"provider":"app\\Models\\Usuarios\\usuarios"} []
[2015-12-22 17:47:57] app.WARNING: Username could not be found in the selected user provider. {"username":null,"provider":"app\\Models\\Usuarios\\usuarios"} []
[2015-12-22 17:49:44] app.WARNING: Username could not be found in the selected user provider. {"username":null,"provider":"app\\Models\\Usuarios\\usuarios"} []
[2015-12-22 17:49:46] app.WARNING: Username could not be found in the selected user provider. {"username":null,"provider":"app\\Models\\Usuarios\\usuarios"} []
[2015-12-22 17:49:54] app.WARNING: Username could not be found in the selected user provider. {"username":null,"provider":"app\\Models\\Usuarios\\usuarios"} []
[2015-12-22 17:51:31] app.WARNING: Username could not be found in the selected user provider. {"username":null,"provider":"app\\Models\\Usuarios\\usuarios"} []
[2015-12-22 17:51:39] app.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException: No route found for "GET /index.html" (from "http://admin.portal.dev/") (uncaught exception) at /var/www/admin-portal/vendor/symfony/http-kernel/EventListener/RouterListener.php line 123 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /index.html\" (from \"http://admin.portal.dev/\") at /var/www/admin-portal/vendor/symfony/http-kernel/EventListener/RouterListener.php:123, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/index.html\". at /var/www/admin-portal/vendor/symfony/routing/Matcher/UrlMatcher.php:98)"} []
[2015-12-22 17:51:50] app.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException: No route found for "GET /login.html" (from "http://admin.portal.dev/") (uncaught exception) at /var/www/admin-portal/vendor/symfony/http-kernel/EventListener/RouterListener.php line 123 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /login.html\" (from \"http://admin.portal.dev/\") at /var/www/admin-portal/vendor/symfony/http-kernel/EventListener/RouterListener.php:123, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/login.html\". at /var/www/admin-portal/vendor/symfony/routing/Matcher/UrlMatcher.php:98)"} []
[2015-12-22 17:52:18] app.WARNING: Username could not be found in the selected user provider. {"username":null,"provider":"app\\Models\\Usuarios\\usuarios"} []
[2015-12-22 17:52:49] app.WARNING: Username could not be found in the selected user provider. {"username":null,"provider":"app\\Models\\Usuarios\\usuarios"} []
[2015-12-22 17:52:54] app.WARNING: Username could not be found in the selected user provider. {"username":null,"provider":"app\\Models\\Usuarios\\usuarios"} []
[2015-12-22 18:00:07] app.WARNING: Username could not be found in the selected user provider. {"username":null,"provider":"app\\Models\\Usuarios\\usuarios"} []
[2015-12-22 18:00:10] app.WARNING: Username could not be found in the selected user provider. {"username":null,"provider":"app\\Models\\Usuarios\\usuarios"} []
[2015-12-22 18:00:30] app.WARNING: Username could not be found in the selected user provider. {"username":null,"provider":"app\\Models\\Usuarios\\usuarios"} []
[2015-12-22 18:00:39] app.WARNING: Username could not be found in the selected user provider. {"username":null,"provider":"app\\Models\\Usuarios\\usuarios"} []
[2015-12-22 18:01:19] app.WARNING: Username could not be found in the selected user provider. {"username":null,"provider":"app\\Models\\Usuarios\\usuarios"} []
[2015-12-22 18:19:43] app.WARNING: Username could not be found in the selected user provider. {"username":null,"provider":"app\\Models\\Usuarios\\usuarios"} []
[2015-12-22 18:27:25] app.WARNING: Username could not be found in the selected user provider. {"username":null,"provider":"app\\Models\\Usuarios\\usuarios"} []
[2015-12-22 18:27:45] app.WARNING: Username could not be found in the selected user provider. {"username":null,"provider":"app\\Models\\Usuarios\\usuarios"} []
[2015-12-22 18:31:27] app.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException: No route found for "GET /index.html" (from "http://admin.portal.dev/") (uncaught exception) at /var/www/admin-portal/vendor/symfony/http-kernel/EventListener/RouterListener.php line 123 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /index.html\" (from \"http://admin.portal.dev/\") at /var/www/admin-portal/vendor/symfony/http-kernel/EventListener/RouterListener.php:123, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/index.html\". at /var/www/admin-portal/vendor/symfony/routing/Matcher/UrlMatcher.php:98)"} []
